### PR TITLE
First attempt

### PR DESCRIPTION
**Pull request by cyberW0MBAT**
I'm aware of using two h1 tags but the other option was to use "< h1 > + < br >" because < h2 > already had some styling which breaks the site if I change it. I couldn't find evidence for a negative effect from using two < h1 >s but I definitely won't do that in the future - just FYI : ))

Please don't sqeeze the viewport under 480 x 806 px 😄 

Looking forward for the feedback.

Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	<nav class="navContainer">
+	  <div>
+	    <a href="#aboutMeSection">About</a>
+	    <a class="contact" href="#contactMeSection">Contact</a>
+	  </div>
+	  <a class="downloadButton" href="#">Download CV</a>
+	</nav>
+
+	<header>
+		<div class="words word-1">
```
Comment (nealioa) [inline index.html:25]: It's an aesthetic and my opinion, but I think the effects applied to these don't look great. Which is why you may have set display: none set in the media query for word-1. If you don't want to keep them then delete them.

**Review by nealioa — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
```
[the remark this answers is not in the document]
Comment (cyberW0MBAT) [reply index.html:2]: Fixed. Thank you

File: style.css [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+}
+
+.downloadButton {
+  float: right;
+  font-size: 1.5em;
+  text-align: center;
+  border-radius: 0.5rem;
+  background-color: transparent;
+  color: black;
+  border: 0.1rem solid black;
```
[the remark this answers is not in the document]
Comment (cyberW0MBAT) [reply style.css:40]: True story... I replaced them all with hex values. Thanks

File: style.css [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+  border-top: 0.3rem #00F5D4 white;
+  background-color: #00BBF9;
+  font-size: 1rem;
+  text-align: center;
+  padding: 0.45rem;
+  margin: 0 auto;
+}
+
+@media screen and (max-width: 60rem) {
+	.innerContainerAboutMe {
```
[the remark this answers is not in the document]
Comment (cyberW0MBAT) [reply style.css:235]: I've missed that. Just fixed it. Thanks

File: index.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  		   </div>
+	  </section>
+
+	<footer class="footerContainer" id="contactMeSection">
+	  <a href="mailto:genadi.kozarev@gmail.com" target="_blank"><img class="icons" src="assets/email-icon.png" alt="Email icon"></a>
+	  <a href="https://github.com/cyberW0MBAT" target="_blank"><img class="icons" src="assets/github-icon.png" alt="GitHub icon"></a>
+	  <a href="https://www.linkedin.com/in/genadi-kozarev/" target="_blank"><img class="icons" src="assets/linkedin-icon.png" alt="LinkedIn icon"></a>
+	</footer> 
+
+	<p class="copyright">Genadi Kozarev ©2021</p>
```
[the remark this answers is not in the document]
Comment (cyberW0MBAT) [reply index.html:121]: Replaced with `&copy`

**Review by norayna — COMMENTED** [review]
approved retroactively

**Pull request merged**